packages/flutter_tools/lib/src/project_validator.dart

```diff
@@ -224,6 +224,7 @@ class GeneralInfoProjectValidator extends ProjectValidator{
       result.add(_materialDesignResult(flutterManifest));
       result.add(_pluginValidatorResult(flutterManifest));
     }
+    result.add(await project.android.validateJavaGradleAgpVersions());
```

Comment:
I think I might need to write an android validator. I couldnt find one.

Reply:
This LGTM